download_from_csv.py

```diff
@@ -44,11 +153,10 @@
         exit()
     print 'Storing files in', folder
 
-
 songs = []
 with open(csvfile, 'rb') as csvfile:
     reader = csv.reader(csvfile)
-    next(reader)  # Skip the first line
+    #next(reader)  # Skip the first line
```

Comment:
In case of generated csv file, first song was being skipped. But now, this case is also handled.